ra/ra_test.go

```diff
@@ -652,24 +637,40 @@ func TestReuseValidAuthorization(t *testing.T) {
 	ra.reuseValidAuthz = true
 
 	// Create one finalized authorization
-	finalAuthz := AuthzInitial
-	finalAuthz.Status = "valid"
 	exp := ra.clk.Now().Add(365 * 24 * time.Hour)
-	finalAuthz.Expires = &exp
-	finalAuthz.Challenges[0].Status = "valid"
-	finalAuthz.Challenges[0].Type = core.ChallengeTypeHTTP01
-	finalAuthz.RegistrationID = Registration.ID
-	finalAuthz, err := sa.NewPendingAuthorization(ctx, finalAuthz)
-	test.AssertNotError(t, err, "Could not store test pending authorization")
-	err = sa.FinalizeAuthorization(ctx, finalAuthz)
-	test.AssertNotError(t, err, "Could not finalize test pending authorization")
+	authz := core.Authorization{
```

Comment:
I think now is a good chance to pull out this common pattern of creating a `core.Authorization`, marshaling it to PB, creating it with `NewAuthorizations2` and then finalizing it with `FinalizeAuthorizations2` into a test helper function. I think there are a handful of places where this flow is duplicated in the `ra_test.go` (here, in `TestReuseAuthorizationDisabled`,`TestReuseExpiringAuthorization`, etc)

Reply:
Good point, I've factored out all of this boilerplate in favor of helper functions here and in the SA tests.